demos/2d_compressible_ALA/ALA_p_nullspace.py

```diff
@@ -0,0 +1,33 @@
+from gadopt import *
```

Comment:
Could you add a comment on top to say what this script does. I mean, I can see it, and it looks useful, but just for people looking at a demo, and quickly want to understand which python file does what... Actually maybe name it 'visualise_ALA_p_nullspace.py' ?

Reply:
I changed it to a demo completely just in case it is useful for anyone, and have copied the math I used as a discussion there.